.evergreen/node-gyp-bug-workaround.sh

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env bash
-
-# This is a workaround for a node-gyp bug that has not fully been investigated
-# due to problems reproducing it outside of CI environments (even though it
-# occurs both in evergreen and github actions).
-# Something seems to go wrong when node-gyp extracts the Node.js header tarball,
-# on Windows specifically (this is most likely because node-tar treats
-# the overwriting of existing files differently on Windows than on other OS --
-# for good reasons, but still).
-# The most likely cause of this issue is that node-gyp somehow extracts the
-# same headers tarball twice, in parallel, in the same location, with race
-# conditions in the tar extraction code leading to issues.
-# The extraction result ends up in %LOCALAPPDATA%\node-gyp\Cache.
-# Manually extracting the tarballs will solve this issue, so we're doing that
-# here.
-# For actually resolving the bug, we would probably need somebody with a local
-# reproduction. However, it seems likely that other people will also encounter
-# this issue, so there's also a good chance that this workaround will just
-# not be needed with a future node-gyp version.
```

Comment:
node-gyp worked on windows without this script, so I'm assuming that it's not needed anymore